RecoMET/Configuration/python/RecoMET_EventContent_cff.py

```diff
@@ -64,6 +64,7 @@
                                            'drop recoHcalNoiseRBXs_*_*_*',
                                            'keep HcalNoiseSummary_hcalnoise_*_*',
                                            'keep *GlobalHaloData_*_*_*',
+                                           'keep *CSCHaloData_*_*_*',
```

Comment:
Please don't use wildcards on the first field.
(the neighboring lines better be updated to, but can be done by someone else in JetMET)